modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,125 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
+| Entity type name | A string obtained from the OData request using #[attributes.odataRequestAttributes], which is specified by the entity data model.
+| Entity type fields | An array of strings obtained from the OData request using #[attributes.odataRequestAttributes], which provides the list of property fields from the entity type.
+| Entity type keys | An object obtained from the OData request using #[attributes.odataRequestAttributes], which provides the map of entity key names and values.
+| System query options | Parameter obtained from the OData request using #[attributes.odataRequestAttributes].
+| Search transformation enabled | Parameter to define the behavior of the transformation.
+| Strict search term match | Parameter to define the behavior of the transformation.
+|====
+
+=== $filter expressions in OData 4 
+
+​​The `$filter` expression in OData uses comparison, logical, arithmetic, and grouping operators. You can access the list operators with their corresponding meaning at http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations[the OData4 Specification]. As mentioned before, APIkit does not support `has` and `divby` operators.
+
+=== Supported expressions by transform to SQL select
+
+OData supports a variety of expressions for filtering that you can find in the [specification](http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations). `transform-to-sql-select` operation supports the following subset of unary and binary operators, along with the following method operators.
+
+==== Binary Operators in OData and in SQL
+
+|====
+| *OData Operator* | *SQL Operator*
+| add | +
+| and | AND
+| div | /
+| eq | =
+| ge | >=
+| gt | >
+| le | =<
+| lt | <
+| mod | %
+| mul | *
+| ne | <>
+| or | OR
+| sub | -
+|====
+
+==== Unary Operators in OData and in SQL
+
+|====
+| *OData Operator* | *SQL Operator*
+| not | NOT 
+| - | - 
+|====
+
+==== Method Operators in OData and in SQL
+
+|====
+| *OData Operator* | *SQL Operator*
+| contains | LIKE %value%
+| startswith | LIKE value%
+| endswith | LIKE %value
+|====
+
+=== $search support
+
+The operation builds a “where” clause based on the `$search` expression provided by the system query option. The implementation translates the search expression to select all the elements that have properties values that match the condition. By default, a term matches if the field value “contains” the term, but it will not match if it is set to be “strict” by configuration.
+
+The following parameters define the `$search` feature:
+
+* Search transformation enabled: it enables (true) or disables (false) the transformation support for this option.
+
+* Strict search term match: it defines if the match should be strict (true) or not strict (false), meaning that the field value should exactly match the term(s) of the search expression.
```

Comment:
These are parameters of the operation too. I would keep the format similar to the way that we described the rest of the parameters in line #140, using a table. What do you think about that?

Reply:
Good catch! Change done! ;)